cmd/podman/images/scp.go

```diff
@@ -0,0 +1,316 @@
+package images
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	urlP "net/url"
+	"os"
+	"strings"
+
+	"github.com/containers/common/pkg/config"
+	"github.com/containers/podman/v3/cmd/podman/common"
+	"github.com/containers/podman/v3/cmd/podman/parse"
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/cmd/podman/system/connection"
+	"github.com/containers/podman/v3/libpod/define"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	scpD "github.com/dtylman/scp"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh"
+)
+
+var (
+	saveScpDescription = `Securely copy an image from one host to another.`
+	imageScpCommand    = &cobra.Command{
+		Use:               "scp [options] IMAGE [IMAGE...]",
```

Comment:
```suggestion
		Annotations:       map[string]string{registry.EngineMode: registry.ABIMode},
		Use:               "scp [options] IMAGE [IMAGE...]",
```
If scp does not support remote adding the annotation will remove `scp` from the remote command.